### ISSUE TITLE
Touch Bar doesn't work when in Exposé

### ISSUE THREAD

**Issue by terry-au** [billziss-gh/EnergyBar#15]
Basically, the toucher is disabled when in Exposé. This is a bit annoying as it means the escape key is does not function.

**Comment by billziss-gh** [comment]
You are correct. The EnergyBar touch bar is visible, but is not responding to touches. I am not sure why this happens.

The workaround to get the ESC key in all cases is to use the FN key.
